index.js

```diff
@@ -135,5 +135,12 @@ module.exports = {
     'react/jsx-one-expression-per-line': ['off'], // <-- This one is a bit annoying
     'no-await-in-loop': ['off'], // <-- Let's disable it and see if it bites us
     'no-underscore-dangle': ['off'], // We use _ in almost all repos. See discussion here: https://github.com/transloadit/botty/pull/41#discussion_r738022104
+
+    'no-implicit-coercion': [
+      'error',
```

Comment:
We'll need to trial this as a warning before projects can upgrade, and once there are no more warnings, then we promote to error.

Reply:
sgtm

Reply:
Counter proposal: I think we should keep it as an error, projects should override that rule to be a warning instead until they've fixed all the occurrences. (please no more warnings 🥺)